android/src/main/java/org/gooddollar/FaceTecModule.java

```diff
@@ -177,4 +183,29 @@ public void faceVerification(String enrollmentIdentifier,
         lastProcessor = processor;
         processor.enroll(enrollmentIdentifier, maxRetries);
     }
+
+    private FaceTecSDKStatus getSDKStatus(Activity activity) {
+        return FaceTecSDK.getStatus(activity);
+    }
+
+    private FaceTecSDK.InitializeCallback onInitializationAttempt(final Activity activity, final Promise promise) {
+        return new FaceTecSDK.InitializeCallback() {
+            @Override
+            public void onCompletion(final boolean successful) {
+                if(successful) {
+                    FaceTecSDK.setDynamicStrings(Customization.UITextStrings);
+                    promise.resolve(true);
+                    return;
+                }
+
+                FaceTecSDKStatus sdkStatus = getSDKStatus(activity);
+                String customMessage = null;
+                if (sdkStatus == FaceTecSDKStatus.NEVER_INITIALIZED) {
```

Comment:
1. make it multi-line like on web & IOS
2. replace word "Simulator" to "Android Emulator"